### ISSUE TITLE
[Item] Spell damage/healing boosting relics

### ISSUE THREAD

**Issue by rawrimalion** [FeenixServerProject/Archangel_2.4.3_Bugtracker#1049]
Seems like every single relic that's supposed to boost some spell's damage/healing doesn't work. Tested all rare level 70 librams available, none of them work. Have a feeling that druid/shaman relics with similar effects don't work either, so feel free to test and add something to this topic.
1. Blessed Book of Nagrand:
   Item ID: 25644
   Old tooltip: http://old.wowhead.com/spell=32403

Has no effect on flash of light, same healing with and w/o it.
1. Libram of the Eternal Rest:
   Item ID: 27917
   Old tooltip: http://www.wowwiki.com/Libram_of_the_Eternal_Rest

Has no effect on concecration's damage, same ticks with and w/o it.
1. Libram of the Lightbringer
   Item ID: 28296
   Old tooltip: http://db.hellground.pl/?item=28296

Same healing of holy light with and w/o it, on all ranks, tested many times.

Basically, there is not a single good working libram for prot/holy paladin out there, which is very sad. Fixing them would be great, thanks.

Kind regards
